src/lightning_app/storage/drive.py

```diff
@@ -34,6 +34,13 @@ def __init__(
                 When not provided, it is automatically inferred by Lightning.
             root_folder: This is the folder from where the Drive perceives the data (e.g this acts as a mount dir).
         """
+        if id.startswith("s3://"):
```

Comment:
This isn't required as it would break lower. '/' isn't allowed in drive entirely.

Reply:
it's only included to let people know that they can't use s3 buckets in this thing called a `Drive`; it's a product AC since we expect user's might not know which one to use when.